### ISSUE TITLE
inline attribute comments are missing from output

### ISSUE THREAD

**Issue by chadwhitacre** [sphinx-doc/sphinx#2006]
Inline attribute doc-comments (#431) were implemented in https://github.com/sphinx-doc/sphinx/commit/a63230f12d1f5af6de24a59ba12c17cc8943cdcd and are in [the Changelog](http://sphinx-doc.org/changes.html) under "Release 1.1 (Oct 9, 2011)." Over at https://github.com/rtfd/readthedocs.org/issues/632 we seem to be finding that under Sphinx 1.3.1 this feature has gone missing. Wherefore art thou, inline attribute comments?
## Steps to Reproduce
1. create and activate a Python 2.7 virtualenv
2. `pip install sphinx` (current version is 1.3.1)
3. `git clone https://github.com/gratipay/filesystem_tree.py.git`
4. `cd filesystem_tree`
5. `python setup.py install`
6. `cd docs`
7. `make html`
8. `cd _build/html`
9. `python -m SimpleHTTPServer`
10. Browse to http://localhost:8000/
## Expected Result

Documentation of [these class attributes](https://github.com/gratipay/filesystem_tree.py/blob/2c5c07e9388d58c6683cc522a0d7681a31712f84/filesystem_tree.py#L97-L100) prior to the documentation for [`mk`](https://github.com/gratipay/filesystem_tree.py/blob/2c5c07e9388d58c6683cc522a0d7681a31712f84/filesystem_tree.py#L133).
## Actual Result

The class attributes are missing:

![screen shot 2015-08-17 at 1 14 14 pm](https://cloud.githubusercontent.com/assets/134455/9311404/e18da7a2-44e3-11e5-81f0-d4979d55383f.png)

**Comment by tk0miya** [comment]
It seems your package is installed as egg package. As a result, autodoc failed to read the source code. I got following debug log:
```
[autodoc] module analyzer failed: source file is not present: '/usr/local/lib/python3.5/dist-packages/filesystem_tree-1.1.2.dev0-py3.5.egg/filesystem_tree.py'
```

Note: I used following Dockefile:
```
FROM tk0miya/sphinx-html

RUN git clone https://github.com/gratipay/filesystem_tree.py.git
WORKDIR /docs/filesystem_tree.py
RUN pip3 install -U pip setuptools
RUN python3 setup.py install
RUN make -C docs html SPHINXOPTS=-vvv
```

As a workaround, you can avoid it by `pip install .`.

**Comment by tk0miya** [comment]
I noticed this issue was already resolved in Sphinx-1.8 (refs: #5290 and #5392). I confirmed that new Sphinx surely generates a document for `FilesystemTree.prefix` from its comment.
Closing.
